### ISSUE TITLE
Elevated plasma Isovalery1/2mebutyrylcarnitine, C5:0

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5961]
**Preferred term label:** Elevated plasma Isovalery1/2mebutyrylcarnitine, C5:0

**Synonyms**

**Definition (free text, please give PubMed ID)** Elevated Isovalery1/2mebutyrylcarnitine concentration in the blood., PMID: 2310400

**Parent term (use hpo.jax.org/app)** Abnormal circulating acetylcarnitine concentration HP:0012071

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
